<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineSend</class>
 <widget class="QWidget" name="LineSend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>25</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LineSend</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,0">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QLineEdit" name="lineEdit_label"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_cmd"/>
   </item>
   <item>
    <widget class="QToolButton" name="tBtn">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
